src/crimson/os/seastore/cache.h

```diff
@@ -550,6 +558,12 @@ class Cache {
    */
   CachedExtent::list dirty;
 
+  struct {
+    std::map<Transaction::src_t, uint64_t> trans_created;
```

Comment:
i'd prefer `vector<>` if possible.

Reply:
Do you think `unordered_map<>` is faster for the hot-path to increase the counter indexed by the related labels?

Reply:
changed to `unordered_map<>`

Reply:
i think the distribution of `src_t` won't be very sparse. actually, they are but consecutive integers, so i think vector is much more spacial-efficient than a red-black tree.

Reply:
Yeah, to index consecutive integers, vector is better. I'll change trans_created/committed to vectors.

Reply:
see https://github.com/ceph/ceph/pull/42202/commits/295c5c34ad3573667b6c4e136376708ea07bffff